mesonbuild/modules/python.py

```diff
@@ -536,6 +536,11 @@ def extension_module_method(self, args: T.List['TYPE_var'], kwargs: 'TYPE_kwargs
             pydep = self._dependency_method_impl({})
             if not pydep.found():
                 raise mesonlib.MesonException('Python dependency not found')
+            # On macOS and some Linux distros (Debian) distutils doesn't link
```

Comment:
I copied this comment from a few lines above, however, I'm not sure this statement is accurate. Judging from the `python.pc` distributed with Python, it is never necessary to link with libpython https://github.com/python/cpython/blob/81f7359f67a7166d57a10a3d5366406d9c85f1de/Misc/python.pc.in#L12 on systems where pkg-config is used, which I assume is at least all variants of Linux. The `python.pc.in` does not inserting anything for the `Libs` entry.